kin-core/src/androidTest/java/kin/core/KinClientIntegrationTest.java

```diff
@@ -310,6 +310,30 @@ public void backupRestore() throws Exception {
         }
     }
 
+    @Test
+    public void importAccount_AddOnlyIfNotExists() throws Exception {
+        KinAccount kinAccount = kinClient.addAccount();
+        String uuid = UUID.randomUUID().toString();
```

Comment:
I would use the const `PASSPHRASE`, took me a while to understands what's the param (or change the var name to passphrase)

Reply:
I've copied what you did.. you don't have `PASSPHRASE`.

Reply:
yes, I see where it comes from, I did a loop and there I've used uuid. shame on me.
BTW, did you run those tests manually? CI can't run instrumentation tests for now.

Reply:
yup manually.

Reply:
Just a little question.
After this change, will the current account be the last account in the list of accounts?
Meaning if , for example, developer will do this:
kinClient.getAccount(kinClient.accountCount - 1)
Then will he get this account?